styles/asciidoc.atom-text-editor.less

```diff
@@ -59,7 +59,12 @@ atom-text-editor, :host {
       }
 
       &.heading.blocktitle,
-      &.substitution {
+      &.substitution{
+        color: @syntax-text-color-unobtrusive;
+      }
+      &.meta.attrlist {
+        font-weight: normal;
+        font-style: normal;
```

Comment:
I was already thinking the same thing, but the current fix was quick and dirty. :wink:  I will improve it.